src/Microsoft.ML.Tokenizers/Model/Tiktoken.cs

```diff
@@ -94,7 +94,7 @@ public sealed class Tiktoken : Model
                     string s = Encoding.UTF8.GetString(kvp.Key.ToArray());
 
                     // Don't add mal-formed utf8 converted bytes to the vocab.
-                    if (!StringContainInvalidChars(s))
+                    if (!s.Contains('\uFFFD'))
```

Comment:
Is this going to use LINQ's Contains when targeting netstandard2.0? If yes, it should use IndexOf() >= 0 instead.